activation/post_verifier_test.go

```diff
@@ -115,12 +115,17 @@ func TestPostVerifierPrioritization(t *testing.T) {
 	require.NoError(t, err)
 
 	verifier.EXPECT().
-		Verify(context.WithValue(context.Background(), prioritizedVerifyCall, true),
-			gomock.Any(), &shared.ProofMetadata{NodeId: nodeID.Bytes()}, gomock.Any()).
+		Verify(
+			context.Background(),
+			gomock.Any(),
+			&shared.ProofMetadata{}, gomock.Any()).
 		Return(nil)
 
-	err = v.Verify(context.WithValue(context.Background(), prioritizedVerifyCall, true),
-		&shared.Proof{}, &shared.ProofMetadata{NodeId: nodeID.Bytes()})
+	err = v.Verify(
+		context.Background(),
+		&shared.Proof{},
+		&shared.ProofMetadata{},
+		PrioritizedCall())
```

Comment:
does the test fail without the assertion that `ProofMetadata` contains `nodeID`?

Reply:
No, but in `develop` branch behavior is same in already existing test